frontends/smvparser.y

```diff
@@ -714,7 +695,7 @@ array_type: arrayword sizev of type_identifier{
 
 sizev:
     "[" integer_val "]"{
-        $$  = $2;
+        $$  = stoi($2);
```

Comment:
Just to check my understanding, is this converted to an integer because it'll be used as the size parameter for bit-vectors?